Assets/Scripts/Game/Guilds/GuildManager.cs

```diff
@@ -266,7 +266,8 @@ public FactionFile.GuildGroups GetGuildGroup(int factionId)
                 guildGroup = (FactionFile.GuildGroups)factionData.ggroup;
 
                 // Handle temples nested under deity
-                if (factionData.children != null && (guildGroup == FactionFile.GuildGroups.None && factionData.children.Count > 0))
+                if (factionData.type == (int)FactionFile.FactionTypes.God
```

Comment:
I wanted to look through the history of this one. Other than a comment change, this part of code has been relatively untouched since 2018.

![image](https://user-images.githubusercontent.com/10426244/175185901-2dadb554-996d-4fcb-bfd8-1143c60d54e8.png)

Looking at the methods which call this one, I can't see any immediate problems with limiting this to just `FactionTypes.God`. That seems to be the intent of this conditional from the start.

I'll quickly tag @ajrb for feedback in case we're making an error here. Hazelnut wrote this and might recall if there are issues limiting to just `FactionTypes.God` child factions.

If nobody raises any issues, I'll go ahead and merge.

Reply:
Pretty sure this will be okay. It was only temples that would have children and no guild on the faction id so I never thought to limit it just to god factions. As long as temple guilds still work then this change is fine I think.

Reply:
Thank you Hazel! I'll run through some tests with temple guilds and merge if no problems found.